README.md

```diff
@@ -1,29 +1,83 @@
+## Understanding Tuples
 
-## Outline/ Notes
+You are already familiar with arrays, which allow storing an ordered list of values of the same type. Those can be iterated over using a for-loop, like in this example with the list of seasons:
 
-* Start with the known- students know how to iterate over arrays. Show an example of iterating over an array. 
-    * Present some real world problem such as, “If you have a dictionary where the keys represent the seasons and the values represent your favorite color associated with that season, is there an easy way to print out a list that for each sentence and color that says “In the {season} my favorite color is {color}.”
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
-    
+```swift
+let seasons = ["Winter", "Spring", "Summer", "Fall"]
+
+for season in seasons {
+  print(season) // e.g. Winter
+}
+```
+
+In this exercise, we will learn about another type which allows grouping multiple, related values, the *tuple*. A tuple can contain values of any type where each value can be of a different type. However, a tuple is not considered a collection, therefore it is not possible to iterate over its values with a loop.
+
+For example, you may want to store name and age of a person in a single compound value:
+
+```swift
+let person = ("Frank", 32)
+```
+
+There are multiple ways of accessing the individual values of a tuple. Using index numbers starting with zero:
+
+```swift
+print(person.0) // Frank
+print(person.1) // 32
 ```
+
+By assigning the individual values to new variables:
+
+```swift
+let (name, age) = person
+
+print(name) // Frank
+print(age) // 32
+```
+
+**Note** The purpose of tuples is storing related values together temporarily. If you want more complex structures or longer term storage, classes or structs are more suitable. You will learn about them later.
+
+Tuples can also be used to return multiple values from a function, for example:
+
+```swift
+func fetchPerson(name: String) -> (age: Int, job: String) {
+  let age = 32
+  let job = "Janitor"
+
+  return (age, job)
+}
+
+let name = "Frank"
+let person = fetchPerson(name)
+print("\(name) is a \(person.job) and \(person.age) years old.") // Frank is a Janitor and 32 years old.
+```
+
+In this case, we are using a *named tuple*. This makes it obvious what each value means without having to read the whole function. However, named tuples are not limited to functions, you can create them anywhere in your program:
```

Comment:
Before jumping right into using what a named tuple is. Get the student thinking first by asking a question, is there an easier way to decompose the elements of a tuple without having to resort to using .0 or .1 to access the first and second elements of a tuple. Then stating, obviously if there was a way to name these individual elements (which you did above), then it would much easier to utilize that tuple. By stating this, it's much more clear to the student WHY they should name them like you did. It signals to them that they are about to learn something new and they are primed for it.